### PR TITLE
fix: update the final decision icon in the studentcard

### LINKED ISSUE #661: final decision icon update
**Describe the bug**
small icon to indicate final decision does not get updated. This is espacially confusing if you are changing the final decision

**To Reproduce**
1. open a detailed screen for a user
2. set/change the final decision

**Expected behavior**
this updated immediately

**Screenshots**

https://user-images.githubusercontent.com/68779933/169057922-760e1a1d-c9fe-4a0b-9e04-c976aeea32aa.mov

### PR DESCRIPTION
**Pull request by Mouwrice**
closes #661

### CODE REVIEW

File: frontend/components/StudentOverview/StudentOverview.tsx [hunk 1]
```diff
@@ -36,8 +36,10 @@ export const StudentOverview: React.FC<{
     const { getSession, isAdmin } = useContext(SessionContext);
     const { notify } = useContext(NotificationContext);
     const { socket } = useSockets();
+    const [studentcard, setStudentcard] = useState<Student>(student);
 
     const fetchEvals = async () => {
+        console.log("fetching");
```
Comment (jay-tux) [inline frontend/components/StudentOverview/StudentOverview.tsx:42]: 🚨 The console.log police is here 🚨

**Review by jay-tux — CHANGES_REQUESTED** [review]
Watch out! The police is here!

**Review by jay-tux — APPROVED** [review]
🔝

**Review by BramDevlaminck — APPROVED** [review]
awesome!

**Pull request merged**